nss-day-cohort-51/nss-day-cohort-51.github.io
# Mw feat add video icon learn more 102

# Description
On the DevCard, I added the FAVideo icon to the LEARN MORE button.
Merged with main and resolved conflicts;
Importing the react-icon items to the MediaButton component changes after accepting both changes

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Testing Instructions
- Fetch branch and run in browser
- View DevCard, and the video icon should display on the Learn More button; it should also be the same color as the text including on hover 

ADDITION:
Added a "More Details" button on the Event Card.
Added styling to fall in-line with existing buttons.

# Checklist:
